blog/2020-11-19-project-droplet-closing-notes.md

```diff
@@ -0,0 +1,187 @@
+      "assetcommitment": "0ad37...",
+      "commitmentnonce": "02c64...",
+      "commitmentnonce_fully_valid": true,
+      "n": 0,
+      "scriptPubKey": {
+        "asm": "OP_HASH160 24f69b6b10f3e71b69a130bcd8ffaf3a69624305 OP_EQUAL",
+        "hex": "a91424f69b6b10f3e71b69a130bcd8ffaf3a6962430587",
+        "reqSigs": 1,
+        "type": "scripthash",
+        "addresses": [
+          "XEigbcVAKLpzMoRhPXCHFRVw7Gs5FsRjQP"
+        ]
+      }
+ }
+
+```
+
+#### Confidential transactions in a nutshell
+
+To build a confidential transaction the receiver needs to create a *blinded address* which is a regular address with an additional public key attached to it. This public key is called the *blinding key*. 
+The creator of the transaction uses this blinding key to hide the assets and amounts of transaction outputs.
+
+To hide amounts and asset types Elements uses Pedersen commitments, a commitment scheme which lets you keep a piece of data secret but commit to it so that you cannot change it later. 
+This may sound similar to hash functions; Pedersen commitments go one step further though and allow for an additional property: commitments can be added up, the sum of the commitment of data data is the same as the commitment of the sums data, e.g. : `C(bf_1, d_1) + C(bf_2, d_2) == C(bf_1 + bf_2, d_1 + d_2)` where `bf_1` and `bf_2` are blinding factors and `d_1` and `d_2` are data .
+
+For hiding amounts in Elements, our Pedersen commitments are constructed using elliptic curve (EC) points.
+EC public keys obey the additively homomorphic property as described above. 
+With this tool we can replace the integer-based amount (8 bytes) with a 33-byte Pedersen commitment.
+
+Pedersen commitments are not enough, we also need rangeproofs and surjection proofs to ensure that the hidden amounts are within a certain range. 
+A more detailed explanation would go beyond the scope of this blog post. 
+We refer the interested reader to [[8]](https://elementsproject.org/features/confidential-transactions/investigation) and our PoC in [[3]](https://github.com/comit-network/droplet).
+
+### Native assets
+
+Another core feature of Elements is the ability to issue assets. 
+Every asset is identified by an `Asset ID` and has one (or a federation of multiple) issuer. 
+A list of all currently available assets on Elements can be found here [[7]](https://blockstream.info/liquid/assets). 
+In our opinion the most interesting assets on Liquid are L-BTC - a token pegged 1-to-1 to the Bitcoin mainchain - and L-USDT - a USD stable coin issued by Tether.
+
+When transferring any asset the sender has to specify the `asset` ID she wishes to transfer.
+
+Note: Transaction fees on Liquid are always paid in L-BTC. 
+However, one can use a third party such as Liquid Taxi [[5]](https://liquid.taxi/) who will pay for your transaction fees in L-BTC. 
+In return you pay the service provider in a different asset.
+
+A non-confidential UTXO can look like this:
```

Comment:
This line seems out-of-context, talking about asset and then presenting a non-confidential UTXO. Is the intent to present a multi-asset UTXO instead?

Reply:
The intention was to show the `asset` field int he utxo.... let me think about it and rephrase.